.github/workflows/ci.yml

```diff
@@ -61,7 +59,17 @@ jobs:
           arch: ${{ matrix.arch }}
       - uses: julia-actions/cache@v2
       - uses: julia-actions/julia-buildpkg@v1
+      - uses: dcarbone/install-jq-action@v3
+        if: runner.os == 'Linux' || runner.os == 'Windows'
+      - name: On Linux and Windows, add Github.com known hosts (from GitHub API over HTTPS)
+        shell: bash
+        run: |
+          mkdir -p ~/.ssh
+          curl --silent https://api.github.com/meta | jq --raw-output '"github.com "+.ssh_keys[]' >> ~/.ssh/known_hosts
+        if: runner.os == 'Linux' || runner.os == 'Windows'
```

Comment:
Isn't [`ssh-keyscan github.com >> ~/.ssh/known_hosts`](https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/45acaa5c78a091ef75e5daa5de7a8676600509a5/.github/workflows/ci.yml#L118C11-L118C55) sufficient?

Reply:
Done in 472fdffadfb605a02c7e43485bb935213a00b6e3